node_modules/oae-core/termsandconditions/js/termsandconditions.js

```diff
@@ -21,6 +21,27 @@ define(['jquery', 'oae.core'], function($, oae) {
         var $rootel = $('#' + uid);
 
         /**
+         * Accept the invitation in the URL state, if any
+         */
+        var acceptInvitation = function() {
+            var invitationToken = oae.api.util.url().param('invitationToken');
```

Comment:
As of the Hilary ticket that was merged, there is no way to accept an invitation and not result in a verified email address.